lib/project_types/script/layers/infrastructure/test_suite_repository.rb

```diff
@@ -23,18 +23,6 @@ def create_test_suite(script)
           write_aspect_type_definitions_file
         end
 
-        def get_test_suite(language, extension_point_type, script_name)
```

Comment:
Why is this file not deleted entirely?

Reply:
Because we use the rest of this file to create the test suite. After my other refactors I'll be changing that and removing this though!